schemas/context/viewability-details.schema.json

```diff
@@ -0,0 +1,102 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/viewability-details",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Viewability Details",
+  "type": "object",
+  "description": "Browser or app settings measured at runtime describing the viewability of an advertisement or content associated with each page load.",
+  "definitions": {
+    "viewabilityDetails": {
+      "properties": {
+        "xdm:adUnitDepth": {
+          "title": "Ad Unit Depth",
+          "type": "integer",
+          "description":
+          "Publishers may embed ad units inside containers/iFrames in order limit the ad's access solely to the code of the page. This describes how many 'containers' or iFrames the ad unit is displayed inside of.",
+          "minimum": 0
+        },
+        "xdm:viewportHeight": {
+          "title": "Viewport Height",
+          "type": "integer",
+          "description":
+          "The number of vertical pixels of the window the ad is displayed in, measured at runtime.",
+          "minimum": 0
+
+        },
+        "xdm:viewportWidth": {
+          "title": "Viewport Width",
+          "type": "integer",
+          "description":
+          "The number of horizontal pixels of the window the ad is displayed in, measured at runtime.",
```

Comment:
What about non-pixel-based media?

Reply:
IAB and MRC guidelines are only defined for pixel-based media. Following industry standard guidelines for consistency.

Reply:
For viewport measurement, the browser standard is to always return a pixel count by definition. This is analogous to window.innerHeight (https://developer.mozilla.org/en-US/docs/Web/API/Window/innerHeight)

Reply:
The correct standard is the CSS View module, which identifies the measurement as CSS Pixels (https://www.w3.org/TR/cssom-view-1/#css-pixels).  So let's go ahead and use that correct terminology and then I'm good. Thanks!